File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map<String, Map<String, Integer>> typerCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(
        String license,
        Account driver,
        Map<String, Map<String, Integer>> typerCarAccepted,
        ArrayList<String> seatsMaterial,
        ){
            super(license, driver);
            this.typerCarAccepted = typerCarAccepted;
            this.seatsMaterial = seatsMaterial;
    }
}

File: Python/UberBlack.py
from car import Car 

class UberBlack(Car):
    typerCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typerCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial